app/src/main/java/campus/tech/kakao/contacts/ContactListAdapter.kt

```diff
@@ -0,0 +1,52 @@
+package campus.tech.kakao.contacts
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import campus.tech.kakao.contacts.model.Contact
+
+class ContactListAdapter(
+    private val inflater: LayoutInflater,
+    val context: Context,
+    private val contactList: MutableList<Contact>,
+    private val listener: (view: View, index: Int) -> Unit = { _, _ -> }
+) :
+    RecyclerView.Adapter<ContactListAdapter.ContactViewHolder>() {
+    class ContactViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private val textInsideImage: TextView
+        private val contactName: TextView
+        fun bind(contact: Contact) {
+            contactName.text = contact.name
+            textInsideImage.text = (contact.name).getOrNull(0).toString()
```

Comment:
```suggestion
            textInsideImage.text = (contact.name).firstOrNull().toString()
```
getOrNull(0) 보단 firstOrNull()이 좀더 직관적으로 표현되는것 같네요 :)

Reply:
수정하였습니다!